spec/spec_helper.rb

```diff
@@ -38,6 +38,13 @@
   config.default_sleep_interval = 0.33
   config.clear_lets_on_failure = true
   config.default_retry_count = 3
+
+  unless ENV['FULL_BACKTRACE']
```

Comment:
I got tired of seeing Rails' stack trace for every failed spec. This was much faster/easier to see why tests were failing.